sources/packages/backend/apps/db-migrations/src/sql/SFASPartTimeApplications/Add-col-py-maximum-support.sql

```diff
@@ -0,0 +1,39 @@
+-- Add column cspt_award for SFAS Part Time Application.
+ALTER TABLE 
+  sims.sfas_part_time_applications 
+ADD 
+  COLUMN IF NOT EXISTS cspt_award NUMERIC(10);
```

Comment:
Not expecting to be part of PR. But as FYI @andrewsignori-aot @guru-aot @andrepestana-aot @sh16011993 

We are using different data types for amount between sfas_individuals and here. 

![image](https://github.com/bcgov/SIMS/assets/54600590/747a1f6d-f672-412b-a38b-9fe65b696b6a)

I know we have a ticket to fix it.

Reply:
Yes as @dheepak-aot mentioned, we should follow the numeric(8,2). Please change it accordingly.

Reply:
@dheepak-aot @guru-aot @andrewsignori-aot The amount data types the ones in this PR are int(10) on the document "SFAS to SIMS Bridge", and “0000004000” is for $4000.
![image](https://github.com/bcgov/SIMS/assets/148148914/0fc6bbc8-8129-4c0f-8643-ae1fc9ec3045)
If we change it to numerica(8, 2), then “0000004000” will probably become $40.00.

Reply:
Hi @lewischen-aot there seem to be a difference in the way how awards are read in `sfas_applications(full time)` and `sfas_part_time_applications(part_time)` 

The SFAS full time records are considering decimal at last 2 positions but Part time records are not considering decimals.
**Full time:**
![image](https://github.com/bcgov/SIMS/assets/54600590/aaae8eb7-2b46-4c25-9b25-e8c9fa6c2450)


**Part time:**
![image](https://github.com/bcgov/SIMS/assets/54600590/a6ea4e8d-1a65-4bc6-a55b-14db1b9caa8d)

Due to this reason, please continue to follow the same approach for data-type and reading values for SFAS_PART_TIME_APPLICATIONS.

And please raise this question to business and @JasonCTang on this difference to validate the document.